scripts/stablex/get_auction_elements.js

```diff
@@ -85,8 +85,7 @@ const printOrder = function(order, currentBatchId) {
 module.exports = async callback => {
   try {
     const instance = await BatchExchange.deployed()
-    const auctionElementsEncoded = await instance.getEncodedOrders.call()
-    let auctionElementsDecoded = decodeOrdersBN(auctionElementsEncoded)
+    let auctionElementsDecoded = await getOrdersViaPaginatedApproach(instance, 100)
```

Comment:
nit: maybe make this an optional argv option as well with 100 being the default?